ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -390,26 +390,7 @@ struct FrameworkBuilder {
       } catch {
         fatalError("Error while enumerating files \(headersDir): \(error.localizedDescription)")
       }
-      // Verify Firebase frameworks include an explicit umbrella header for Firebase.h.
```

Comment:
Update the zip distribution to use the standard CocoaPods generated umbrella header.

Going forward we can encourage usage of module imports instead of the manually constructed Firebase umbrella header which won't exist once the libraries are fully converted to Swift.

Reply:
Should this be a Firebase 9 thing, or what effect does this have now?

Reply:
Recommending modules is a Firebase 9 thing. We've never needed this check for the zip build since CocoaPods builds its own umbrella header. This is a check that should have been a formatting check for repo structure.

BTW, FirebaseFunctions.h going away is another breaking change for the API review.